DashAI/back/config_object.py

```diff
@@ -16,3 +17,7 @@ def get_schema(cls) -> dict:
             Dictionary representing the Json Schema of the component.
         """
         return cls.SCHEMA.model_json_schema()
+
+    def validate_and_transform(self, raw_schema: dict) -> dict:
```

Comment:
Why not call the data `raw_data` instead of raw_schema?
```suggestion
    def validate_and_transform(self, raw_data: dict) -> dict:
```